File: Geek Assistant (Android AIO)/Modules/General/Action/GA_infoAsk.vb
Module GA_infoAsk
    Public infoAnswer As Boolean = False
    Public infoButtonText As (LeftButton As String, RightButton As String)
    ''' <summary>
    ''' Uses My.Forms.info to ask the user something + allows custom buttons text
    ''' </summary>
    ''' <param name="Title">Title of the question</param>
    ''' <param name="body">More text to view below the title</param>
    ''' <param name="YesButton">Text of the Left YesButton</param>
    ''' <param name="NoButton">Text of the Right NoButton</param> 
    ''' <returns>True if YesButton was clicked or False if NoButton was clicked</returns>
    Public Function Run(Title As String, body As String, YesButton As String, NoButton As String,
                        Optional IconLight As Image = Nothing,
                        Optional IconDark As Image = Nothing,
                        Optional TextColorLight As Object = Nothing,
                        Optional TextColorDark As Object = Nothing) As Boolean
        If String.IsNullOrEmpty(NoButton) Then Return False
        If String.IsNullOrEmpty(YesButton) Then Return False
        If String.IsNullOrEmpty(body) Then Return False
        If String.IsNullOrEmpty(Title) Then Return False
        If IconDark IsNot Nothing Then info.info_IconDark = IconDark
        If IconLight IsNot Nothing Then info.info_IconLight = IconLight
        If TextColorLight IsNot Nothing Then info.info_TextColorLight = TextColorLight
        If TextColorDark IsNot Nothing Then info.info_TextColorDark = TextColorDark

        infoAnswer = False
        infoButtonText = (YesButton, NoButton)
        My.Settings.info_MsgLevel = 2 'Question
        My.Settings.info_MsgTitle = Title
        My.Settings.info_Msg = body
        info.ShowDialog()
        Return infoAnswer
    End Function
End Module

File: Geek Assistant (Android AIO)/Modules/General/Action/GA_infoAsk.cs

using GeekAssistant.Forms;
using System;
using System.Drawing;

internal static partial class GA_infoAsk {
    public static bool infoAnswer = false;
    public static (string LeftButton, string RightButton) infoButtonText;
    /// <summary>
    /// Uses My.Forms.info to ask the user something + allows custom buttons text
    /// </summary>
    /// <param name="Title">Title of the question</param>
    /// <param name="body">More text to view below the title</param>
    /// <param name="YesButton">Text of the Left YesButton</param>
    /// <param name="NoButton">Text of the Right NoButton</param>
    /// <returns>True if YesButton was clicked or False if NoButton was clicked</returns>
    public static bool Run(string Title,
                           string body,
                           string YesButton, string NoButton,
                           Image IconLight= null,
                           Image IconDark = null,
                           Color ? TextColorLight = null,
                           Color ? TextColorDark = null) {
        c.ErrorInfo.code = $"{txt.GA_GetErrorInitials()}-null-"; 
        try {
            if (string.IsNullOrEmpty(Title))
                c.ErrorInfo.code += "t"; 
            if (string.IsNullOrEmpty(body))
                c.ErrorInfo.code += "b"; 
            if (string.IsNullOrEmpty(YesButton))
                c.ErrorInfo.code += "y"; 
            if (string.IsNullOrEmpty(NoButton))
                c.ErrorInfo.code += "n";
            if (c.ErrorInfo.code != $"{txt.GA_GetErrorInitials()}-null-") throw new ArgumentException();
        } catch (ArgumentException ex) {
            GA_Msg.DoMsg(10, "Something went wrong\nWe encountered a problem while trying to ask you something...",2,ex.ToString());
        } 
        if (IconDark != null) c.Info().info_IconDark = IconDark;
        if (IconLight != null) c.Info().info_IconLight = IconLight;
        if (TextColorLight != Color.Empty) c.Info().info_TextColorLight = TextColorLight ?? Color.Empty;
        if (TextColorDark != Color.Empty) c.Info().info_TextColorDark = TextColorDark ?? Color.Empty;


        infoAnswer = false;
        infoButtonText = (YesButton, NoButton);
        c.S.info_MsgLevel = 2; // Question
        c.S.info_MsgTitle = Title;
        c.S.info_Msg = body;
        c.Info().ShowDialog();
        return infoAnswer;
    } 
}